sdk/src/androidTest/java/com/karhoo/sdk/api/VerifyQuoteIntegrationTest.kt

```diff
@@ -1,12 +1,12 @@
-package com.karhoo.sdk.api.util
+package com.karhoo.sdk.api
 
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import com.github.tomakehurst.wiremock.junit.WireMockRule
-import com.karhoo.sdk.api.KarhooApi
-import com.karhoo.sdk.api.KarhooError
 import com.karhoo.sdk.api.model.Quote
 import com.karhoo.sdk.api.network.response.Resource
 import com.karhoo.sdk.api.testrunner.SDKTestConfig
+import com.karhoo.sdk.api.util.ServerRobot
```

Comment:
Duplicated

Reply:
Tried fixing this already Android Studio throws errors if I try to remove these

Reply:
Both server robots here are different, not duplicated